src/hooks/keywordSearch.test.ts

```diff
@@ -0,0 +1,59 @@
+import Cookies from 'js-cookie';
+import nock from 'nock';
+import { StatusCodes } from 'http-status-codes';
+import { List } from 'immutable';
+import { buildGetItemsByKeywordRoute } from '../api/routes';
+import { mockHook, setUpTest } from '../../test/utils';
+import { ITEMS, UNAUTHORIZED_RESPONSE } from '../../test/constants';
+import { buildSearchByKeywordKey } from '../config/keys';
+import { Item } from '../types';
+
+const { hooks, wrapper, queryClient } = setUpTest();
+jest.spyOn(Cookies, 'get').mockReturnValue({ session: 'somesession' });
+
+describe('Keyword Search Hook', () => {
+  afterEach(() => {
+    nock.cleanAll();
+    queryClient.clear();
+  });
+
+  describe('useKeywordSearch', () => {
+    const range = 'all';
```

Comment:
Factor out, use an enum (hopefully later we will have a common repo to reuse the enums..)